android/app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,6 @@
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
     <!-- Remove some unused RN default permissions -->
     <uses-permission tools:node="remove" android:name="android.permission.READ_PHONE_STATE" />
-    <uses-permission tools:node="remove" android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
Just commenting for now only because this PR is titled "adding back permission" and all I see here is removing a permission. Just want to be sure this is the intention.

Reply:
"tools:node='remove'" was removing the permission, removing the line adds it back as it is a default added by RN.

Reply:
oh, i do know the solution and it's schema based links. https will embed, textile-dev: will break out. that is why i added that accept page in http so that if a user lands there it will textile: schema link them and they can break out.

Reply:
^ for ios. for android it was easier.

Reply:
Ah got it. Thx.